--- mssilk-1.0.0/configure.ac.orig	2013-06-12 16:41:28.000000000 +0200
+++ mssilk-1.0.0/configure.ac	2013-07-07 14:05:26.418714300 +0200
@@ -3,7 +3,6 @@
 
 AC_INIT([mssilk],[1.0.0])
 AC_PREREQ(2.63)
-AC_CONFIG_SRCDIR([sdk/known_checksums/SILK_SDK_SRC_v1.0.8.zip.md5])
 AC_CONFIG_AUX_DIR([build-aux])
 AC_CONFIG_MACRO_DIR([m4])
 AM_INIT_AUTOMAKE([tar-ustar --warnings=no-portability])
@@ -25,59 +25,6 @@
 
 AC_SUBST(STRICT_CFLAGS)
 
-case $host in
-	arm*)
-		SILK_FLAVOUR=ARM
-	;;
-	*)
-		SILK_FLAVOUR=FIX
-	;;
-esac
-
-case $host in
-        armv6-apple*)
-                SILK_MAKE_OPTIONS="ADDED_DEFINES+=IPHONE TOOLCHAIN_PREFIX=XXXX"
-		CFLAGS="$CFLAGS -mno-thumb"
-        ;;
-        armv7-apple*)
-                SILK_MAKE_OPTIONS="USE_NEON=yes ADDED_DEFINES+=IPHONE TOOLCHAIN_PREFIX=XXXX"
-		CFLAGS="$CFLAGS -mno-thumb"
-        ;;
-  *)
-     ;;
-esac
-
-AC_SUBST([SILK_FLAVOUR])
-AC_SUBST([SILK_MAKE_OPTIONS])
-
-# Checks for programs.
-AC_PATH_PROG([WGET],[wget])
-if test -z "$WGET"; then
-   AC_MSG_ERROR(['wget' is required to build this package])
-fi
-
-AC_PATH_PROG([MD5SUM],[md5sum])
-if test -z "$MD5SUM"; then
-	AC_PATH_PROG([MD5SUM],[openssl])
-	if test -n "$MD5SUM"; then
-		MD5SUM="$MD5SUM md5"
-	fi
-fi
-
-if test -z "$MD5SUM"; then
-   AC_MSG_ERROR(['md5sum' is required to build this package])
-fi
-
-AC_PATH_PROG([UNZIP],[unzip])
-if test -z "$UNZIP"; then
-   AC_MSG_ERROR(['unzip' is required to build this package])
-fi
-
-AC_PATH_PROG([PATCH],[patch])
-if test -z "$PATCH"; then
-   AC_MSG_ERROR(['patch' is required to build this package])
-fi
-
 
 LT_INIT([disable-static])
 
--- mssilk-1.0.0/Makefile.am.orig	2013-06-12 16:41:28.000000000 +0200
+++ mssilk-1.0.0/Makefile.am	2013-07-07 14:06:19.492045409 +0200
@@ -1,9 +1,4 @@
 EXTRA_DIST=autogen.sh mssilk.iss.in 
-SILK_VERSION = 1.0.9
-#SILK_DIR=$(srcdir)/sdk/SILK_SDK_SRC_v$(SILK_VERSION)/SILK_SDK_SRC_$(SILK_FLAVOUR)_v$(SILK_VERSION)
-SILK_DIR=$(CURDIR)/sdk/SILK_SDK_SRC_v$(SILK_VERSION)/SILK_SDK_SRC_$(SILK_FLAVOUR)_v$(SILK_VERSION)
-
-SUBDIRS = sdk
 
 
 pluginsdir = $(libdir)/mediastreamer/plugins
@@ -13,7 +8,7 @@ plugins_LTLIBRARIES=libmssilk.la
 libmssilk_la_SOURCES=silk_dec.c silk_enc.c
 
 
-SILK_LIBS = -L${SILK_DIR} -lSKP_SILK_SDK
+SILK_LIBS = -lSKP_SILK_SDK
 libmssilk_la_LIBADD=\
 	$(ORTP_LIBS) \
         $(MEDIASTREAMER_LIBS) \
@@ -21,7 +16,7 @@ libmssilk_la_LIBADD=\
 
 libmssilk_la_LDFLAGS=-module -no-undefined
 
-SILK_CFLAGS = -I$(SILK_DIR)/interface 
+SILK_CFLAGS =
 AM_CFLAGS=      $(SILK_CFLAGS) \
                 $(MEDIASTREAMER_CFLAGS) \
 				$(STRICT_CFLAGS)
