diff -urNp -x '*.orig' mssilk-1.1.1.org/Makefile.am mssilk-1.1.1/Makefile.am
--- mssilk-1.1.1.org/Makefile.am	2016-03-17 15:03:40.000000000 +0100
+++ mssilk-1.1.1/Makefile.am	2021-07-06 00:26:34.986591109 +0200
@@ -1,9 +1,4 @@
 EXTRA_DIST=autogen.sh mssilk.iss.in CMakeLists.txt
-SILK_VERSION = 1.0.9
-#SILK_DIR=$(srcdir)/sdk/SILK_SDK_SRC_v$(SILK_VERSION)/SILK_SDK_SRC_$(SILK_FLAVOUR)_v$(SILK_VERSION)
-SILK_DIR=$(top_builddir)/sdk/SILK_SDK_SRC_v$(SILK_VERSION)/SILK_SDK_SRC_$(SILK_FLAVOUR)_v$(SILK_VERSION)
-
-SUBDIRS = sdk
 
 
 pluginsdir = $(libdir)/mediastreamer/plugins
@@ -13,7 +8,7 @@ plugins_LTLIBRARIES=libmssilk.la
 libmssilk_la_SOURCES=silk_dec.c silk_enc.c
 
 
-SILK_LIBS = -L${SILK_DIR} -lSKP_SILK_SDK
+SILK_LIBS = -lSKP_SILK_SDK
 libmssilk_la_LIBADD=\
 	$(ORTP_LIBS) \
         $(MEDIASTREAMER_LIBS) \
@@ -21,7 +16,7 @@ libmssilk_la_LIBADD=\
 
 libmssilk_la_LDFLAGS=-module -no-undefined
 
-SILK_CFLAGS = -I$(SILK_DIR)/interface 
+SILK_CFLAGS =
 AM_CFLAGS=      $(SILK_CFLAGS) \
                 $(MEDIASTREAMER_CFLAGS) \
 				$(STRICT_CFLAGS)
diff -urNp -x '*.orig' mssilk-1.1.1.org/configure.ac mssilk-1.1.1/configure.ac
--- mssilk-1.1.1.org/configure.ac	2017-07-20 10:47:56.000000000 +0200
+++ mssilk-1.1.1/configure.ac	2021-07-06 00:26:34.986591109 +0200
@@ -3,7 +3,6 @@
 
 AC_INIT([mssilk],[1.1.0])
 AC_PREREQ(2.63)
-AC_CONFIG_SRCDIR([sdk/known_checksums/SILK_SDK_SRC_v1.0.8.zip.md5])
 AC_CONFIG_MACRO_DIR([m4])
 AM_INIT_AUTOMAKE([tar-ustar --warnings=no-portability])
 
@@ -26,48 +25,6 @@ fi
 
 AC_SUBST(STRICT_CFLAGS)
 
-case $host in
-	arm*)
-		case $CC in
-			*clang*) # clang doesn't like the old ARM assembly
-				SILK_FLAVOUR=FIX
-			;;
-			*)
-				SILK_FLAVOUR=ARM
-			;;
-		esac
-	;;
-	*)
-		SILK_FLAVOUR=FIX
-	;;
-esac
-
-case $host in
-	armv6-apple*)
-		SILK_MAKE_OPTIONS="ADDED_DEFINES+=IPHONE TOOLCHAIN_PREFIX=XXXX"
-		CFLAGS="$CFLAGS -mno-thumb"
-	;;
-	armv7-apple*)
-		SILK_MAKE_OPTIONS="USE_NEON=yes ADDED_DEFINES+=IPHONE TOOLCHAIN_PREFIX=XXXX"
-		CFLAGS="$CFLAGS -mno-thumb"
-	;;
-	armv7*)
-		SILK_MAKE_OPTIONS="USE_NEON=yes TOOLCHAIN_PREFIX=XXXX"
-	;;
-	arm*)
-		dnl override flavour for case of old arm, as a workaround of a bug in the Skype's Makefile resulting in neon functions not being compiled while requested,
-		dnl because the compiler says it supports armv7 and neon.
-		SILK_FLAVOUR=FIX
-	;;
-	*)
-	;;
-esac
-
-
-
-AC_SUBST([SILK_FLAVOUR])
-AC_SUBST([SILK_MAKE_OPTIONS])
-
 LT_INIT([disable-static])
 
 AC_ARG_WITH([mediastreamer], AS_HELP_STRING([--without-mediastreamer], [Permits to deactivate mediastreamer check for download of SILK code only]))
