--- mssilk-1.0.1/configure.ac.orig	2015-07-30 22:24:28.056188806 +0200
+++ mssilk-1.0.1/configure.ac	2015-07-31 15:33:33.943597578 +0200
@@ -3,7 +3,6 @@
 
 AC_INIT([mssilk],[1.0.1])
 AC_PREREQ(2.63)
-AC_CONFIG_SRCDIR([sdk/known_checksums/SILK_SDK_SRC_v1.0.8.zip.md5])
 AC_CONFIG_MACRO_DIR([m4])
 AM_INIT_AUTOMAKE([tar-ustar --warnings=no-portability])
 
@@ -26,42 +25,6 @@
 
 AC_SUBST(STRICT_CFLAGS)
 
-case $host in
-	arm*)
-		case $CC in
-			*clang*) # clang doesn't like the old ARM assembly
-				SILK_FLAVOUR=FIX
-			;;
-			*)
-				SILK_FLAVOUR=ARM
-			;;
-		esac
-	;;
-	*)
-		SILK_FLAVOUR=FIX
-	;;
-esac
-
-case $host in
-	armv6-apple*)
-		SILK_MAKE_OPTIONS="ADDED_DEFINES+=IPHONE TOOLCHAIN_PREFIX=XXXX"
-		CFLAGS="$CFLAGS -mno-thumb"
-	;;
-	armv7-apple*)
-		SILK_MAKE_OPTIONS="USE_NEON=yes ADDED_DEFINES+=IPHONE TOOLCHAIN_PREFIX=XXXX"
-		CFLAGS="$CFLAGS -mno-thumb"
-	;;
-	armv7*)
-		SILK_MAKE_OPTIONS="USE_NEON=yes TOOLCHAIN_PREFIX=XXXX"
-	;;
-*)
-	;;
-esac
-
-
-
-AC_SUBST([SILK_FLAVOUR])
-AC_SUBST([SILK_MAKE_OPTIONS])
 
 LT_INIT([disable-static])
 
--- mssilk-1.0.1/Makefile.am.orig	2015-02-17 14:17:43.000000000 +0100
+++ mssilk-1.0.1/Makefile.am	2015-07-31 15:34:38.080261553 +0200
@@ -1,9 +1,4 @@
 EXTRA_DIST=autogen.sh mssilk.iss.in 
-SILK_VERSION = 1.0.9
-#SILK_DIR=$(srcdir)/sdk/SILK_SDK_SRC_v$(SILK_VERSION)/SILK_SDK_SRC_$(SILK_FLAVOUR)_v$(SILK_VERSION)
-SILK_DIR=$(top_builddir)/sdk/SILK_SDK_SRC_v$(SILK_VERSION)/SILK_SDK_SRC_$(SILK_FLAVOUR)_v$(SILK_VERSION)
-
-SUBDIRS = sdk
 
 
 pluginsdir = $(libdir)/mediastreamer/plugins
@@ -13,7 +8,7 @@
 libmssilk_la_SOURCES=silk_dec.c silk_enc.c
 
 
-SILK_LIBS = -L${SILK_DIR} -lSKP_SILK_SDK
+SILK_LIBS = -lSKP_SILK_SDK
 libmssilk_la_LIBADD=\
 	$(ORTP_LIBS) \
         $(MEDIASTREAMER_LIBS) \
@@ -21,7 +16,7 @@
 
 libmssilk_la_LDFLAGS=-module -no-undefined
 
-SILK_CFLAGS = -I$(SILK_DIR)/interface 
+SILK_CFLAGS =
 AM_CFLAGS=      $(SILK_CFLAGS) \
                 $(MEDIASTREAMER_CFLAGS) \
 				$(STRICT_CFLAGS)
